packages/bootstrap/scss/pivotgrid/_theme.scss

```diff
@@ -1,4 +1,6 @@
-@include exports("pivotgrid/theme") {
+@import "~@progress/kendo-theme-default/scss/pivotgrid/_theme.scss";
+
+@include exports("pivotgrid/theme/legacy") {
```

Comment:
why are we keeping the legacy styling?

Reply:
Due to the fact that jQuery still uses it and if we merge the PR before they change their rendering with the new one, it will broke their widget. Once they apply the new rendering, we can safely remove the legacy modules.

Reply:
Also, they will have both renderings for a while. So, we need the styles for now.